power_grid_model_io_native_c/power_grid_model_io_native_c/src/pgm_vnf_converter.cpp

```diff
@@ -37,16 +37,16 @@ PGM_IO_VnfPgmConverter* PGM_IO_create_vnf_converter(PGM_IO_Handle* handle, char
             default:
                 throw power_grid_model::MissingCaseForEnumError{"PGM_IO_create_vnf_converter", experimental_features};
             }
-            auto* converter = new PGM_IO_VnfPgmConverter(file_buffer, experimental_feature);
+            auto* converter = new PGM_IO_PgmVnfConverter(file_buffer, experimental_feature);
             parse_vnf_file_wrapper(converter);
             return converter;
         },
         PGM_IO_regular_error);
 }
 
-char const* PGM_IO_vnf_pgm_converter_get_input_data(PGM_IO_Handle* handle, PGM_IO_VnfPgmConverter* converter_ptr) {
+char const* PGM_IO_pgm_vnf_converter_get_input_data(PGM_IO_Handle* handle, PGM_IO_PgmVnfConverter* converter_ptr) {
     return call_with_catch(
         handle, [converter_ptr] { return convert_input_wrapper(converter_ptr).c_str(); }, PGM_IO_regular_error);
 }
 
-void PGM_IO_destroy_vnf_converter(PGM_IO_VnfPgmConverter* converter_ptr) { delete converter_ptr; }
+void PGM_IO_destroy_pgm_vnf_converter(PGM_IO_PgmVnfConverter* converter_ptr) { delete converter_ptr; }
```

Comment:
We didn't really discuss it specifically for C api, but we discussed it in general. The general decision was that pgm should be in front because pgm is the creator and holder of the converter.

Reply:
in that case, i'd consider that `PGM_IO_vnf_converter_*` is sufficient (due to the prefix `PGM_IO_*`), but what i'm slightly worried about is that it's specifically VNF -> PGM format. compare e.g. the serializer vs the deserializer. that's 2 different classes for which one goes in one direction, and another in the other.

Reply:
I can see why `PGM_IO_vnf_converter_*` would be enough. 

And for worries about VNF -> PGM, won't it be the final goal to at some point in the future have VNF <-> PGM? Then it would still make sense to have PGM in front.

Reply:
just like serializer/deserializer, that would probably be a separate class... but i don't particularly mind either way. @TonyXiang8787 what do you think?

Reply:
I do not have a preference